recipes/libusb/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "libusb" %}
+{% set version = "1.0.22" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: 'https://github.com/libusb/libusb/releases/download/v{{ version}}/{{ name }}-{{ version }}.tar.bz2'
+  sha256: 75aeb9d59a4fdb800d329a545c2e6799f732362193b465ea198f2aa275518157
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('libusb') }}
```

Comment:
Currently looks like the SONAME is changing in the next release. ( https://github.com/libusb/libusb/issues/602 )

Reply:
I don't know what implications that would have, as I'm still struggling to understand how `pin_subpackage` works :S

Reply:
Resolved via e6516d8

Reply:
It means that packages that link with 1.0.22 may be broken when 1.0.23 is released. That said, I don't know if upstream is committed to that SONAME change. So we should just stay aware and be prepared once that release lands. Alternatively we could pin things really tightly in the short term and relax the pinning after that release.